ui: fix various bugs on db console schedules page

kyle-a-wong opened the pull request:
- Updated max result size of sql query to fetch schedules
  to ensure all schedules are rendered in
  db console
- Fixed a bug where changing the "show" and "status" values
  in the url query params resulted in a "something went wrong"
  page. Now, if an invalid value is set in the "show" or "status"
  query parameters, the corresponding dropdown will reset to the
  default value. ("Show: Latest 50" and "Status: All" respectively)

Fixes: #143925, #143924
Epic: None
Release note (bug fix): Fixes bugs in the db console page:
- Previously, the schedules page only showed a subset of the total
  schedules for a cluster due to a missing parameter in the server api
  call. Now the schedules table should correctly show all schedules in
  a cluster
- Fixed a bug where manually upadting the "show" or "status" query

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/144638)
<!-- Reviewable:end -->

kyle-a-wong committed:
ui: fix various bugs on db console schedules page

- Updated max result size of sql query to fetch schedules
  to ensure all schedules are rendered in
  db console
- Fixed a bug where changing the "show" and "status" values
  in the url query params resulted in a "something went wrong"
  page. Now, if an invalid value is set in the "show" or "status"
  query parameters, the corresponding dropdown will reset to the
  default value. ("Show: Latest 50" and "Status: All" respectively)

Fixes: CRDB-49163, CRDB-49164
Epic: None
Release note (bug fix): Fixes bugs in the db console page:
- Previously, the schedules page only showed a subset of the total
  schedules for a cluster due to a missing parameter in the server api
  call. Now the schedules table should correctly show all schedules in
  a cluster
- Fixed a bug where manually upadting the "show" or "status" query

kyle-a-wong commented:
tftr!

bors r+
